.dockerignore

```diff
@@ -0,0 +1,2 @@
+data/
+
```

Comment:
This excludes the database from being found by docker when building the image. Docker will try to `stat` this file but it doesn't have read access and this crashes the docker build step otherwise.